### PR TITLE
Mo tro 12

### PR DESCRIPTION
**Pull request by K0LdW0rLd**
#12 
Completed Responsive Header

### CODE REVIEW

File: src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.nav-bar {
+    display: flex;
+    justify-content: flex-end;
+    width: 100vw;
+    margin: 0 auto;
+    background-color: #E5E5E5;
```
Comment (rc1336) [inline src/components/NavBar/NavBar.css]: In the mock the navbar has a white background, when I render it on my computer it is grey. Can we fix that.

Comment (K0LdW0rLd) [reply src/components/NavBar/NavBar.css]: I actually got that background color from the mock.  While it does look white, it says the background color is #E5E5E5

**Comment by K0LdW0rLd** [comment]
Added padding to the top of the mobile version

File: src/components/NavBar/NavBar.css (hunk 1 above)
Comment (rc1336) [reply src/components/NavBar/NavBar.css]: You sir are right the mock is calling for that color. Will have to touch base with the designer, but for now lets keep what the mock says.

Comment (rc1336) [reply src/components/NavBar/NavBar.css]: You sir are right the mock is calling for that color. Will have to touch base with the designer, but for now lets keep what the mock says.

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <nav className="nav-bar">
+                <div className="mobile-style">
+                    {/* Replace with burger component when complete */}
+                    <Link to="#">
+                        <Icons name={['fas', 'bars']} size="2x" />
+                    </Link>
+                </div>
+                <div className="mobile-style">
+                    {/* Replace with logo component */}
+                    <img className="mobile-logo" src={logo} alt="Modern trousseau logo" />
```
Comment (rc1336) [inline src/components/NavBar/NavBar.js]: Can you please import the logo from Component Library here instead of the one you are using.

Comment (K0LdW0rLd) [reply src/components/NavBar/NavBar.js]: Imported logo from Component Library

**Pull request merged**